docker-compose.yml

```diff
@@ -21,7 +17,10 @@ x-sentry-defaults: &sentry_defaults
     - snuba-replacer
     - symbolicator
     - kafka
+  entrypoint: "/etc/sentry/entrypoint.sh"
```

Comment:
This line override `cron/Dockerfile` 
ENTRYPOINT ["/entrypoint.sh"]
As result this service
```
  sentry-cleanup:
    <<: *sentry_defaults
    image: sentry-cleanup-onpremise-local
    build:
      context: ./cron
      args:
        BASE_IMAGE: "$SENTRY_IMAGE"
    command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
```
does not work anymore

Reply:
Ugh, terrible oversight from my end. Apologies. Will see how we can fix this and also catch the issue.